src/types/runtime.go

```diff
@@ -34,6 +34,8 @@ type ZarfInitOptions struct {
 	Components string `json:"components" jsonschema:"description=Comma separated list of optional components to deploy"`
 
 	StorageClass string `json:"storageClass" jsonschema:"description=StorageClass of the k8s cluster Zarf is initializing"`
+
```

Comment:
make sure you run `make docs-and-schema` and commit any changes when modifying types. Our `types` pkg  is used to generate the Typescript defs, JSON Schema & docs.

Reply:
Sure @jeff-mccoy. I will do that.

Reply:
Changes are done in https://github.com/defenseunicorns/zarf/pull/1191/commits/ba3281bba2b85d66b1d322cdb5c6c02edc449f8c